db_name not defined

if I do not use proxy then I see error db_name not defined.
With v1.3.5 it work fine.

Why there done() and then() after it?
https://github.com/nolanlawson/pouchdb-load/blob/2edb5f6652fc171cc7935fbfd6324e913cca2835/lib/index.js#L49

https://github.com/nolanlawson/pouchdb-load/blob/2edb5f6652fc171cc7935fbfd6324e913cca2835/lib/index.js#L56
info is undefined if no proxy

@nolanlawson I just ran across this issue and here is what I've found:
The current tests throw this error, but the tests pass even though the error is thrown.  You can duplicate this by running the tests in chrome with the developer console open.  I see the following trace in the console
```
test-bundle.js:16731 Error: Missing/invalid DB name
    at test-bundle.js:16696
    at test-bundle.js:16729
    at new PouchDB (test-bundle.js:16679)
    at test-bundle.js:55
```

I'll take a look at the tests to see if I can change them to fail when an exception is thrown and then put together a PR to fix the issue.
I think the fix is just to add the following here:
https://github.com/nolanlawson/pouchdb-load/blob/master/lib/index.js#L54
```javascript
if (!opts.proxy) {
  return;
}
```

--- a/lib/index.js
+++ b/lib/index.js
@@ -51,6 +51,9 @@ function loadString(db, data, opts, callback) {
 
     return db.info();
   }).then(function (info) {
+    if (!opts.proxy) {
+      return;
+    }
     var src = new db.constructor(opts.proxy,
       utils.extend(true, {}, {}, opts));
     var target = new db.constructor(info.db_name,

